livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -141,6 +144,9 @@ def __init__(
         api_key = api_key or os.environ.get("ELEVEN_API_KEY")
         if not api_key:
             raise ValueError("ELEVEN_API_KEY must be set")
+        
+        if language_code is not None and model != "eleven_turbo_v2_5":
+            raise ValueError("language_code is only supported for the 'eleven_turbo_v2_5' model")
```

Comment:
It would be better to leave this to the ElevenLabs API. What is its behavior right now if you send language_code on a different model? Is it ignored, or is it a 4xx error?

Reply:
It throws a 400 error. Want me to just log a warning message instead? Also, should I disclude language_code if model != eleven_turbo_v2_5 in the api call then?

Reply:
set it to None in this case and logged a warning